pkg/config/config.go

```diff
@@ -23,7 +23,11 @@ type Feed struct {
 	// Format is "300ms", "1.5h" or "2h45m".
 	// Valid time units are "ns", "us" (or "µs"), "ms", "s", "m", "h".
 	// NOTE: too often update check might drain your API token.
+	// todo deprecated, remove that someday
 	UpdatePeriod Duration `toml:"update_period"`
```

Comment:
As cron expressions are far more advanced, how do you feel if we leave `update_period` as is (no deprecation) as default option to use? I think users will use it in the majority of cases.
```go
if cron != "" {
 // use cron
} else if update_period != "" {
 // use update_period
} else {
 // use default value
}
```

This way we also keep backward compatibility.

Reply:
In my opinion, this is not the best solution. We introduce new entities into the config and into the code that duplicate each other's behavior.

I specifically added a new value so that users have time to update they configs. I think it won't be difficult for them to replace "12h" with "@every 12h".

Do you really want save two variable?

Reply:
The problem is that people don't follow the commit history and usually just do `docker pull + restart`, and expect things to work. I think we should guaranty some degree of compatibility.  Also cron expressions is not a thing that will be used by everyone (though it's still good to have this flexibility).

Reply:
Ok. I think warning message in log and auto-replacement helped them in transition time ;)
But i no have problem with several config variables